modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+[id='virt-troubleshooting-vm-import_{context}']
+= Troubleshooting a virtual machine import
+
+[id='logs_{context}']
+== Logs
+
+ifdef::virt-importing-vmware-vm[]
+You can check the V2V Conversion Pod log for errors.
+
+.Procedure
+
+. Obtain the V2V Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the V2V Conversion Pod log:
```

Comment:
Both of these steps should end with `by running the following command:`

Reply:
OK, grudgingly. I think we over-use this phrase because the IBM Style Guide is based on the premise that users use a UI more than they use the command line. I don't think our users need to be told that they have to run a command when the command is in front of them. No one is ever going to ask us, "Where is the `oc get pods` button?" :-)